wire/wire.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package basictracer_go.wire;
```

Comment:
this is a Dapper-ish impl though, not general purpose. E.g wouldn't work for Zipkin.